plugin/src/main/java/com/playmonumenta/redissync/commands/PlayerHistory.java

```diff
@@ -15,15 +15,14 @@
 
 public class PlayerHistory {
 	public static void register(Plugin plugin) {
+		EntitySelectorArgument.OnePlayer playerArg = new EntitySelectorArgument.OnePlayer("player");
+
 		new CommandAPICommand("playerhistory")
-			.withArguments(new EntitySelectorArgument.OnePlayer("player"))
+			.withArguments(playerArg)
 			.withPermission(CommandPermission.fromString("monumenta.command.playerhistory"))
 			.executes((sender, args) -> {
-				if (!(sender instanceof Player)) {
```

Comment:
Is there a reason for this to be able to execute if sender is not a player?

Reply:
Probably meant to replace it with .executesPlayer which has the same effect, will fix. (Although there's not really a problem with lettings non-players do it since it just sends a message)